google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/facet/AppEngineFacetEditor.java

```diff
@@ -172,6 +174,10 @@ public String getHelpTopic() {
   @Override
   public void onFacetInitialized(@NotNull Facet facet) {
     AppEngineWebIntegration.getInstance().setupDevServer();
+
+    UsageTrackerProvider.getInstance()
+        .trackEvent(GctTracking.APP_ENGINE_ADD_FACET)
```

Comment:
whats the difference between facet addition and facet creation? Is there duplication / overlap?

LGTM otherwise.